pages/service-usages/basic-usages.md

```diff
@@ -0,0 +1,209 @@
+## Basic Usages
+
+[![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/522bf43d2b5f9f09f7ee)
+
+For usage of advanced scenarios, please refer to [Trippin Advanced Usages](service-usages/trippin-advanced-usages/).
+
+### Requesting Data
+
+### Requesting Entity Collections
+
+```
+GET http://services.odata.org/TripPinRESTierService/(S(3mslpb2bc0k5ufk24olpghzx))/People
+```
+
+#### Requesting an Individual Entity by ID
+
+```
+GET http://services.odata.org/TripPinRESTierService/(S(3mslpb2bc0k5ufk24olpghzx))/People('russellwhyte')
+```
+
+#### Requesting Data/Requesting an Individual Property 1/2
+
+```
+GET http://services.odata.org/TripPinRESTierService/(S(3mslpb2bc0k5ufk24olpghzx))/Airports('KSFO')/Name 
+```
+
+```
+GET http://services.odata.org/TripPinRESTierService/(S(3mslpb2bc0k5ufk24olpghzx))/Airports('KSFO')/Location/Address 
+```
+
+#### Requesting an Individual Property Raw Value
+
```

Comment:
Requesting an single primitive or enum type Property Raw Value, we do not support to request structural type property value or collection.